### DISCUSSION TITLE
Github does not recognize changes in the worklows folder

### DISCUSSION THREAD

**Discussion by Jardo-51** [community/community#118710] [Projects and Issues]
### Select Topic Area

Bug

### Body

I added a workflow file into my .github/workflows repository.

In the "Actions" tab, I saw that the workflow file did not get parsed correctly. It said there was a "syntax error at line 10".

I fixed the error and pushed the changes, but the changes did not get recognized. The "Actions" tab still said "syntax error at line 10", and it was showing the old file content.

I checked the "Source" tab in Github, and the file had the correct content.

I tried editing the file directly in Github, but withe same result (Github still shows the old content).

I tried replacing the content of the workflow file with the content from a completely different workflow file (which is 100% correct) but withe same result (Github still shows the old content).

I tried deleting the file, but Github actions still shows it, with the old content.

I tried adding a completely different workflow file (different file name and content), but it is not shown in the "Actions" tab.

I think this is a bug in Github.

**Comment by zehor-l** [comment]
Consider clearing your browser cache, force-pushing changes, checking file encoding, or reaching out to GitHub Support to address the issue
